underlay/src/main/resources/config/criteria/aouSC2023Q3R2_testonly/criteriaselector/variant/selector.json

```diff
@@ -8,8 +8,8 @@
     "tags": null
   },
   "filterBuilder": "core.EntityGroupFilterBuilder",
```

Comment:
Should this be changes to filterableGroupFilterBuilder ? (i assume we create a new one at the backend with this name)

or should this change after the backend change since the service would look for it during start up

Reply:
Yes, once it exists in the backend so the UI can be functional in the interim. It's only referenced in the test underlay for now.